### ISSUE TITLE
Cannot build octocrab - move occurs because `unrecognized` has type xxx

### ISSUE THREAD

**Issue by iamjpotts** [XAMPPRocky/octocrab#446]
Using rustc 1.70 on a Mac, and rustc 1.68 on Ubuntu 20.04, and cloning `main` of this repo, I cannot get the project to build with `cargo b` or `cargo t` - there's a proliferation of errors like this one (almost 100 instances of this error):

```
error[E0507]: cannot move out of `*self` which is behind a shared reference
  --> src/models/activity.rs:16:35
   |
16 | #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
   |                                   ^^^^^^^^^
   |                                   |
   |                                   data moved here
   |                                   move occurs because `unrecognized` has type `Reason`, which does not implement the `Copy` trait
   |
   = note: this error originates in the derive macro `Serialize` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider borrowing here
   |
16 | #[derive(Debug, Clone, PartialEq, &Serialize, Deserialize)]
   |                                   +


```

What am I doing wrong with building `octocrab` ?

What additional information do you need to help me work thru this?

**Comment by BCNelson** [comment]
I'm having this same issue

**Comment by BCNelson** [comment]
This appears to happen on older versions as well. So far i have tested 
- `0.29.3`
- `0.29.2`
- `0.29.1`
- `0.29.0`
- `0.28.0`
- `0.27.0`

**Comment by BCNelson** [comment]
This appears to be a problem with serde `v1.0.184` released two hours ago. Locking serde to version `1.0.183` allows it to compile

**Comment by BCNelson** [comment]
it has been fixed in verison `1.0.185`  #https://github.com/serde-rs/serde/issues/2591

**Comment by iamjpotts** [comment]
I was able to get `octocrab` to build after running `cargo update` this morning.
